### ISSUE TITLE
final draft comments

### ISSUE THREAD

**Issue by maartenmarx** [miguelpieters/miguel-thesis#7]
Hi @miguelpieters ,

hierbij mijn commentaar op de laatste draft. Nog even flink ertegenaan! Succes joh! 
Laat me weten als je hulp nodig hebt.
groet
maarten

# Review final draft thesis miguel

* Marx
* 18 juni

### Algemeen

Ik ben eigenlijk wat geschrokken van je draft, want het is wat mij betreft nog best slordig, en vaak warrig. Er is nu niet veel tijd meer, dus prober het zo goed mogelijk en efficient te repareren.
* wordt heel precies. zorg in iefder geval voor wat formules zodat je laat zien dat je dat echt wel kan.
* herhaal maar niet wat er in tabellen staat in de tekst. 
* Probeer wilde conclusies zo veel mogelijk te vermijden. Hou het heel feitelijk, en probeer alleen dingen te verklaren. Ook dingen die jij voor de hand vind iggen. Geeft niet. 

Dit komt wel goed, maar het is jammer dat je hier niet eerder al meer aandacht aan gegegeven hebt. Je hebt gaaf materiaal, en daar past dus ook een perfecte presentatie bij.

**Nu niet zenuwachtig worden! Effe uitblazen en je tijd gebruiken die je nog hebt. Maar wel echt alle dagen tot zondagavond!**

**Bel me gerust even als je iets naders wilt vragen.**

### aandachtspnten

1. Referenties zijn erg slordig. Kijk hoe Jenny dit doet, en maak het perfect. Er mist vaak wat, hoofdletters zijn kleine letters, ik heb zowel een jaartal als to appear in 1 referenrie, etc. 
2. Intro goed, maar kijk eens naar mijn praatje (per mail gestuurd) om het in een iets breder perspectief te plaatsen. Geef ook goes weer dat jij de tagger ook wilt gebruiken om corpus brede beweringen te doen.
3. 99-101 dit is wel erg kort door de bocht. Je hoofdvraag is heel goed joh!
4. Ik zou sectie 3, *Background* noemen.
5. In 3.3 kan je wel wat meer vertellen. Naive Bayes is ook een statistisch model, maar dan lineair. Dus ga wat dieper in op het spacy model en vertel ook wat over Bert en sciBERT. 
6. Ik zou ook wel graag een sectie 3.4 zien waarin je wat vertelt over specifiek *scientific information extraction*, onder andere dus over dataset extractie. Zie ook mijn praatje.
7. 4.1.3. *The text is split into tokens (words) and sentences using Spacy.*
8. 4.1.4 Ik raak in de war, want je spreekt over "the rule based model" zonder dat er vertelt is hoe dat ding werkt, of wie dat gemaakt heeft, etc.  Als je dat ding van Jenny gebruikt, prima, besteed er een paar zinnen aan en refereer dan aan haar scriptie.
    9. Ik snap het deel 220-232 echt helemaal niet. Hoe weet je of een zin positief of negatief is **voor je gaat annoteren?**
    10. *In total 6.000 sentences will be annotated* verander naar *In total 6.000 sentences were annotated*
11. 4.2 begin. Je moet echt precies definieren wat een exact match en wat een partial match is. Ook zie ik graag de defeinities van P, R en F1.
12. 5.1. Zet de formule van Fleiss kappa er maar in, en geeft exact aan over hoeveel zinnen en hoeveel annotators dit berekent is.
13. 310-311. Wat betekent *missing* precies? Als er een partial overlap is, is die dan ook *missing*? Idem voor *spurious*.
    14. het gebruik van true positives etc, zonder dat daarvoor te noemen is raar. Haal liever weg, en leg gewoon uit wat je bedoelt.
    15. als ik naar tabel 3 kijk lijst het alsof je actual en possible ook kunt definieren als de som van bepaalde voorgaande dingen. Zoek het goed op en doe dat dan heel precies.
    15. l317: nu raak ik de kluts kwijt. Hoe kan je "partial" ook exact meten, en andersom "correct" ook exact? Volgens mij ben jij ook de kluts kwijt.
16. tabel 1: B en I dataset scores zijn niet uitgelegd. Echt doen!!!
17. Figuur 1: ja maar welke maat is dit nu? Partial of exact of nog weer wat anders?
18. Het rare is dat het lijkt alsof de scores in figuur 1 lager zijn dan die in tabel 1: terwijl figuur 1 de training dataen de nog moeilijker zero shot bevat. Hoe kan dat???
19. Het is ook raar dat rule based het zoveel beter doet op zero shot dan op test set. Hoe kan dat?  
20. 331-345 eigenlijk herhaal je wat er ind e tabel staat. Dat hoeft voor mij niet. Liever lees ik duiding en verklaring. Ik ben bang dat hier iets mis is.
21. Eigenlijk bevatten tabellen 1 en 2 exact dezelfde informatie. Kies er eentje uit. Aan de hand van wat je wilt gaan zeggen.
22. refereer naar tables en figures niet met hun titel maar met hun nummertje: Dus Table~\ref{tab:mijn gavelabel}
23. Ik zou secties 5.4.3 en 5.4.2 omdraaien. Dus eerst de nulletjes en dan degene met wel datasets.
24. Hou dus 5.4.3 maar, maar pas het aan:
    25.     zet juist de x-as eens in log, om het lognormale te benadrukken. Je zal dan zien dat dit bij SDM en VISION eigenlijk goed past en dat het vreemde gedrag bij SIGIR en NIPS echt anders is. 
    26. Het enige wat ik kan verzinnen is dat er bij de papers met 1 hit in sigir en NIPS veel false positieven zitten. Zeg de helft: dan zou het ook een meer lognormale curve krijgen. 
        27. Maar dat is maar een hypothese. Ik zou dat gewoon noemen en het zo laten. 
    28. Ik zou dus de grafiek voor alles bij elkaar maar weglaten, want die is zo verwarrend. 
    29. Verander dan je vraag in "How are the number of dataset mentions per article distributed?"
    30. Ik praat hier graag nog wel met je over. 
31. 5.5 maar leg dan wel graag uit waarom de scores nu per jaar zoveel lager zijn dan die in tabel 1.
1. 5.6 zet de y-as op log schaal in figuur 10. En haal 448-451 maar weg. 
    2. Je kan in saeborn mooie scatters maken waarbij je ook de histogrammen van de 2 variabelen naast de assen krijgt, met een kde erbij. Dat is heel inzichtelijk. probeer dat ajb. 
3. Ik zou sectie 6 helemaal weglaten
4. Haal de subsecties in 7 er maar uit en maak daar paragrafen van. Dat loopt lekkerder.
